ui/.env.development

```diff
@@ -1,2 +1,4 @@
-REACT_APP_MIXPANEL_TOKEN=6b8d3fb93ffd04f2e76325540c838624
-PORT=3001
```

Comment:
I suggest keeping this configuration. It is because the local TiDB cluster includes grafana whose default port is 3000 as well. Without this configuration, when we run `yarn start`, it will ask us whether to use another port instead.

```
Would you like to run the app on another port instead? (Y/n)
```

Reply:
fixed